common/logger/infologger/directhook.go

```diff
@@ -38,6 +38,7 @@ import (
 
 	"github.com/AliceO2Group/Control/common/utils"
 	"github.com/sirupsen/logrus"
+	"github.com/spf13/viper"
```

Comment:
The inconvenience here is that you've just pulled viper as a dependency of the executor by way of `DirectHook`, and the executor doesn't even use Viper. I don't know how fat Viper is in absolute terms, but it's a fairly comprehensive library with over a dozen of its own dependencies. So far I've mostly tried to keep the executor small and light as much as possible to ensure fast startup (and even with that, it's up to 28MB now). Most likely it's not a big deal, but it would be good if you could check the effect of this change on binary size, and if significant, try to find a way to avoid the added dependency.

Reply:
Actually, looks like I did that recently by using `utils.TimeTrack` in the executor (and only this so far, AFAICT), so a Viper dependency is already there, unfortunately. I think it would still be good to keep Viper out of the executor, but it's probably out of scope of this logging effort.